taskcluster/ci/test/kind.yml

```diff
@@ -55,7 +55,10 @@ jobs:
         attributes:
             build-type: debug
         treeherder:
-            symbol: ui
+            kind: test
```

Comment:
Nit: No need to specify `kind: test` and `tier: 2` because they're already part of the default configuration up above 🙂